bundle/package.json

```diff
@@ -35,6 +35,7 @@
     "@pixi/filter-adjustment": "^2.5.0",
     "@pixi/filter-advanced-bloom": "^2.5.0",
     "@pixi/filter-ascii": "^2.5.0",
+    "@pixi/filter-bevel": "^2.5.1",
```

Comment:
Small thing, can you make sure the version of the filter is 2.5.0, it will get bumped when we publish the next version.